github/repos_environments.go

```diff
@@ -180,6 +188,39 @@ func (s *RepositoriesService) CreateUpdateEnvironment(ctx context.Context, owner
 	}
 
 	e := new(Environment)
+	resp, err := s.client.Do(ctx, req, e)
+
```

Comment:
Please delete this blank line, as we want to keep the error processing tightly coupled to the line that generates the error.